packages/sources/svelte/src/components/actions/VtmnDropdown/VtmnDropdown.svelte

```diff
@@ -0,0 +1,147 @@
+<script>
+  import { cn } from '../../../utils/classnames';
+  import { createEventDispatcher } from 'svelte';
+  import { clickOutside } from '../../../utils/clickOutside';
+
+  /** @restProps */
+
+  /**
+   * Custom classes to apply to the component.
+   * @type {array} options
+   */
+  export let options;
+
```

Comment:
https://www.decathlon.design/726f8c765/p/42d824-dropdown/b/6647a9

I think, the dropdown component should avec an 'error' export 
![image](https://user-images.githubusercontent.com/2856778/159011734-97f3da96-37f3-4f94-a4f7-faa9808cd8ff.png)

Reply:
Dropdown error css class is depreciated from vitamin css